files/pwm/rerunhostnamestate.sh

```diff
@@ -0,0 +1 @@
+salt-call --local state.apply pwm/hostname
```

Comment:
It's kind of an accident that this syntax works. Recommend using `pwm.hostname` instead.